views/newsletter/templates/blocks/social/block.hbs

```diff
@@ -1,3 +1,7 @@
 <div class="mailpoet_tools"></div>
-<div class="mailpoet_content mailpoet_social"></div>
+<div
+    class="mailpoet_content mailpoet_social"
+    style="{{#ifCond model.styles.block.textAlign '==' 'left'}}text-align:left;{{/ifCond}}{{#ifCond model.styles.block.textAlign '==' 'center'}}text-align:center;{{/ifCond}}{{#ifCond model.styles.block.textAlign '==' 'right'}}text-align:right;{{/ifCond}}"
```

Comment:
This is a very long line, could we do this better? Like, use a class for example? Would that be more maintainable and readable?